backend/tests/test_utils/test_utils.py

```diff
@@ -32,13 +32,14 @@ def test_get_bedrock_client_alt(self):
 
         cli_dict = client.__dict__
 
-        reg = cli_dict["_client_config"].region_name
+        
+
+        reg=cli_dict['_client_config'].region_name
+        
 
-        LOGGER.debug("Region: ")
-        LOGGER.debug(reg)
 
-        assert reg == "us-west-2"
+        reg = cli_dict["_client_config"].region_name
 
 
-if __name__ == "__main__":
```

Comment:
Can I ask the reason why the entry point is removed?

Reply:
I was trying to update the folder name, I may have confused myself with the git history. Do you want to close this PR and just rename it upstream on your end?